tasks/dev.js

```diff
@@ -31,7 +33,12 @@ bundler.on("bundled", async () => {
 });
 
 bundler.on("buildEnd", () => {
-  browserSync.reload();
+  // setTimeout is to avoid browserSync throwing an error
+  // after the IE11 transpile is done 
+  setTimeout(function() {
+    transpileForIE("dist");
```

Comment:
If you comment out this "transpileForIE" call in the dev task, you'll at least have accurate source maps while developing.  Just won't be able to develop using IE.